Reducers/RegistrationReducer.js

```diff
@@ -118,7 +118,7 @@ export default function registrationReducer(state = initialState, action) {
 
         case VALIDATE_REGISTRATION_TELEPHONE: {
             numberNotSet = state.Telephone.trim() === "" ? "Please enter the Phone number" : "";
```

Comment:
We need this trim function before matching the pattern

Reply:
Added trim functions for email and phone validation on RegistrationScreen and email validation on AuthorizationScreen